scylla-macros/src/parser.rs

```diff
@@ -35,6 +35,46 @@ pub(crate) fn parse_named_fields<'a>(
     }
 }
 
+pub(crate) enum StructFields<'a> {
+    Named(&'a FieldsNamed),
+    Unnamed(&'a FieldsUnnamed),
+}
+
+/// Parses the tokens_input to a DeriveInput and returns the struct name from which it derives and
+/// the fields
+pub(crate) fn parse_struct_fields<'a>(
+    input: &'a DeriveInput,
+    current_derive: &str,
+) -> Result<StructFields<'a>, syn::Error> {
```

Comment:
Shouldn't it be:
_Parses the tokens_input **from** a DeriveInput and **~~returns~~** the struct name from which it derives and **returns** the fields_
?

Reply:
Yeah, now that I read the comment it does not make much sense. I copied it from the very similar function which extracts only NAMED fields:
```
/// Parses the tokens_input to a DeriveInput and returns the struct name from which it derives and
/// the named fields
```